internal/worker.go

```diff
@@ -95,6 +102,27 @@ type (
 		// Default value is 2
 		MaxConcurrentDecisionTaskPollers int
 
+		// optional: Sets the minimum number of goroutines that will concurrently poll the
+		// cadence-server to retrieve decision tasks. If FeatureFlags.PollerAutoScalerEnabled is set to true,
+		// changing this value will NOT affect the rate at which the worker is able to consume tasks from a task list.
+		// Default value is 1
+		MinConcurrentDecisionTaskPollers int
+
+		// optional: Sets the interval of poller autoscaling, between which poller autoscaler changes the poller count
+		// based on poll result. It takes effect if FeatureFlags.PollerAutoScalerEnabled is set to true.
+		// Default value is 1 min
```

Comment:
Is 1 minute fast enough?

Reply:
I'm still experimenting on this. Since we ensure at least one poller is always working, the impact of this parameter should be low.